manifests/cf-manifest/manifest/050-rds-broker.yml

```diff
@@ -79,6 +79,25 @@ jobs:
                 supportUrl: "https://forums.aws.amazon.com/forum.jspa?forumID=60"
               plan_updateable: true
               plans:
+                - id: "5f2eec8a-0cad-4ab9-b81e-d6adade2fd42"
+                  name: "Free"
+                  description: "5GB Storage, NOT BACKED UP, Dedicated Instance, Max 200 Concurrent Connections. Postgres Version 9.5. DB Instance Class: db.t2.micro."
+                  free: true
+                  metadata:
+                    costs:
+                      - amount:
+                          usd: 0.02
```

Comment:
Good point.  I'm not sure what the best thing to do here is. This is how much these instances actually cost, but given we're not passing that cost on to the users, maybe this should be omitted.

Having said all that, the only way that a user can see these costs is by using `cf curl`.